requirements/pytorch/strategies.txt

```diff
@@ -2,5 +2,3 @@
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
 deepspeed>=0.6.0, <0.8.0  # TODO: Include 0.8.x after https://github.com/microsoft/DeepSpeed/commit/b587c7e85470329ac25df7c7c2521ff9b2833db7 gets released
-lightning-colossalai>=0.1.0dev1
```

Comment:
If you remove this, then the accelerator connector test for strategy="colossalai" will be skipped. Is this intentional?

Reply:
no, we still install it with GPU CI so from testing there is no change, just these are not forced in dockers... and later we need to find better way for this external strategy testing :chipmunk: